tldw_Server_API/tests/Config/test_config_adapter.py

```
@@ -21,7 +21,7 @@ def test_get_config_section_and_value_reads_file(tmp_path, monkeypatch):
     monkeypatch.setenv("TLDW_CONFIG_FILE", str(cfg))
 
     section = config.get_config_section("Server")
-    assert section["host"] == "0.0.0.0"
+    assert section["host"] == "0.0.0.0"  # nosec B104
```

Ruff (0.15.1)

[error] Possible binding to all interfaces

(S104)